projects/word-weaver-cli/gen-schemas.sh

```diff
@@ -1,20 +1,19 @@
 #!/bin/bash
 
 # Usage:
-#    cd projects/word-weaver-cli
-#    SCHEMA_LANG=lang_code ./gen-schemas.sh
+#    ./gen-schemas.sh <lang_code>
 #
 # Generate JSON schema files for data validation from ../word-weaver/src/config/schema.ts,
 # if SCHEMA_LANG is empty, or ../word-weaver/src/config/schema.$SCHAMA_LANG.ts.
 #
 # E.g.:
-#    DATA_DIR=../word-weaver/src/assets/data/fr/v1 ./compress.sh
+#    ./gen-schemas.sh fr
 
 set -o errexit
 
 CONFIG_PATH=../word-weaver/src/config
-if [[ $SCHEMA_LANG ]]; then
-    CONFIG_FILE=schema."$SCHEMA_LANG".ts
+if [[ $1 ]]; then
+    CONFIG_FILE=schema."$1".ts
```

Comment:
```suggestion
SCHEMA_LANG=$1
if [[ $SCHEMA_LANG ]]; then
    CONFIG_FILE=schema."$SCHEMA_LANG".ts
```
just to make it more self-documenting

Reply:
yep - this is better, go for it!